src/packages/OneUITheme/OneUITheme.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @module OneUITheme
+ */
+import {
+    colorVariables,
+    computedColorVariables,
+    headerFooterVariables,
+    miscVariables,
+    sizingVariables,
+    typographyVariables,
+} from './baseVariables';
+
+export type CSSVars = {
+    [key: string]: string;
+};
```

Comment:
TS supports shorter `Record<string, string>` type for these kind of cases.

Reply:
As far as I know, `Record ` is used for defining a complex structure, and for string keys better to use the index signature. https://betterprogramming.pub/typescripts-record-type-explained-691372b1a449

Reply:
oke